helix_description/urdf/helix_arm.materials.xacro

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" ?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" >
+
+
```

Comment:
This seems to be an empty and unrequired file?

Reply:
Probably need to have a more extensive discussion but what i was thinking was that similar to how it was set up with the separate "dynamixel_block" and "hand" URDFs, we would separate the helix URDF into the top section with the motor assembly (which ive called motor_head, not sure if it's a good name... unless you like heavy metal ;D) and the flexible arm parts. So for now, this particular file is a placeholder since i dont have any description of the arm to add yet.

Reply:
True, I think that's a good setup and having placeholders in place makes also send. 
I understood, thx for your explanations.